ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected-proxy/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-proxy-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-disconnected-proxy-provision
+  steps: 
+  - ref: gcp-provision-vpc
+  - ref: gcp-provision-bastionhost
+  - ref: proxy-config-generate
+  - ref: ipi-conf
+  - ref: ipi-conf-proxy
+  - ref: ipi-conf-gcp
+  - chain: ipi-install
+  env:
+  - name: DISCONNECTED_NETWORK
```

Comment:
Nit: there will be a proxy, so this won't actually be disconnected ;).  Docs prefer "restricted network" (e.g. [here][1]) for "some network traffic might be restricted" without specifying exactly what or how much is restricted.  There are `ipi/conf/aws/blackholenetwork` as well, where nothing gets out of the VPC except traffic that goes through the proxy.  Naming consistency doesn't matter all that much to me, but grepping is easier if we don't come up with too many names for the same things ;).

[1]: https://docs.openshift.com/container-platform/4.10/installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.html

Reply:
Yeah, kinda of agree. Here we want to stress the network is `disconnected`, the same thing like `blackhole network`, the similar reason, we do not want to surprise other people when grepping `blackhole` keyword, so give it a different name.